common/rc

```diff
@@ -333,3 +333,22 @@ unload_module() {
 	done
 	return 1
 }
+
+# Copied from xfstests
```

Comment:
In the "check" script, "grep -A 9999 $dmesg_marker" is used to cut out the dmesg after the marker. I suggest to use same code. This will avoid coping code from xfstests. Could you update the commit to use the grep -A command? (I think it is ok to copy a part of code from xfstests to blktests, but would like to keep such a copy minimum).

Reply:
"grep -A 9999 $dmesg_marker"  sounds good to me.